src/lib/components/primitives/message/input-message.svelte

```diff
@@ -1,11 +1,14 @@
 <script lang="ts">
+  import sanitize from 'xss';
   import type { Maybe } from '$lib/interfaces';
 
   export let message: string;
```

Comment:
I'd like to change the prop's name here from `message` to `dirtyMessage` (I know that sounds silly, but I like being obvious when security is involved). If you use the VS Code "Rename Symbol" function (right click), it should change all references automatically for you.

Reply:
Makes sense.